asset-transfer-private-data/chaincode-go/private_asset_transfer.go

```diff
@@ -0,0 +1,497 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
```

Comment:
General comment about TransientMap values passed in various func: As an application client, it is more error prone to have very different key_names for the json passed in the TransientMap.
for e.g.  currently `DeleteAsset` takes TransientMap key `asset_delete` with a json value, `TransferAsset` takes a key called `asset_owner` with a json containing both owner & buyer_mspid. Naming of the key is misleading, as well.

How about we just simplify all key names to a simple string `args` (all func have only one key) with json value unchanged. For these func  
- CreateAsset
- AgreeToTransfer
- TransferAsset
- DeleteAsset

If above suggest does not fly, then I would request a non-json value in TransientMap like
- DeleteAsset would take key: asset_id > "value" (non json)
- TransferAsset would take keys asset_id > "value" & buyer_msp > "mspvalue"
 Much simpler.